.github/workflows/haskell.yml

```diff
@@ -53,7 +53,7 @@ jobs:
 
     - name: hlint
       run: |
-        cabal install hlint --installdir=dist-newstyle
+        test -f dist-newstyle/hlint || cabal install hlint --installdir=dist-newstyle
```

Comment:
I was getting CI failures because this already existed. I opted not to overwrite it in that case and test for the existence of the binary before installing instead.